brandonfujii/wildhacks-2017-portal
# added reset password endpoint

## Testing Steps
- Pull and run both API and portal
- Go to `/forgot`
- Enter name and send yourself a password recovery email to an address that's associated with an account 
- Follow the link in the email, changing the hostname to your localhost and port.
- Set a new password
- Login again

Worth noting that the animated header DOM element obstructs a lot of real estate on pages that are not the landing page. Might be a good idea to either add a global padding onto these pages or minimize the header. 

We might want to later add better feedback for this, like some success feedback to show that the something has been sent, or more descriptive server errors.

Also added a class for caching in local storage in `/app`. Should abstract a lot of the JSON parsing and stringifying if we decide to cache actual objects rather than just strings.

@thehandsomepanther
